<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Masukkan Data Penumpang</string>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nama Penumpang</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tambahPenumpangButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tambah Penumpang</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pesanButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>56</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Pesan</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="errorMessageLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>35</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
